doc/release-notes/7961-boolean-field-type.md

```diff
@@ -0,0 +1,5 @@
+Boolean support added for fieldType definition. Accepted values are "true", "1", "yes", "Y", "On", "false", "0", "no", "N", "Off". All values are case insensitive.
```

Comment:
@jeromeroucou please see this new pull request where @Asbjoedt has a field that can be yes, no, or unknown:

- #10626

What do you think?

Reply:
Thanks @pdurbin for your comment.

I agree with the discussion in the issue, it repeats what was said in the chat (Zulip), and that by indicating that the new `archival.tsv` file is experimental, I would take its modification into this PR (it will be the first demonstrator of the feature).

Do you agree with me?

Reply:
@jeromeroucou first, sorry for the slow reply. I was on vacation and I'm still catching up!

I'm a little confused. Are you saying you would modify this PR to all for "unknown"? And then we'd use `archival.tsv` to test it?

Reply:
Hi @pdurbin , I hope you vacation was great !

I wanted to say that I think the proposal made in the archival.tsv file seems to correspond to what was exchanged in zulip, i.e. either "yes", "no" or "unknown". And I would modify my PR accordingly to take into account what is proposed in the archival.tsv file. As for the order, I think that the PR #10626 can be processed first, and that I would take it into account via a merge.

Is it more clearly?

Reply:
Well...

The thing about #10626 is that it's using the type "text". I'm referring to the submitToArchivalAppraisal field specifically.

This PR (#10480) adds a new type called "boolean". If we merge this PR first we can have submitToArchivalAppraisal use "boolean" instead, right?